nodejs/FatBeaconPeripheral/FatBeacon.js

```diff
@@ -0,0 +1,50 @@
+var bleno = require('bleno');
+var eddystoneBeacon = require('eddystone-beacon');
+
+/***********************Altering Library Functions**************************/
+/**
+ * In order to broadcast in the fatbeacon format, we override several
+ * functions in the advertisement-data module of the eddystone-beacon library.
+ */
+var AdvertisementData = 
+        require('eddystone-beacon/lib/util/advertisement-data');
+var Eir = require('eddystone-beacon/lib/util/eir');
+
+var FAT_BEACON_FRAME_TYPE = 0x0e;
+var MAX_URL_LENGTH = 18;
+var SERVICE_UUID = 'feaa';
+
+AdvertisementData.makeEirData = function(serviceData) {
+  var eir = new Eir();
+  eir.addFlags(0x06);
+  eir.add16BitCompleteServiceList([SERVICE_UUID]);
+  eir.addServiceData(SERVICE_UUID, serviceData);
+  return eir.buffer();
+}
+
+AdvertisementData.makeUrlBuffer = function(name) {
+  console.log(`AdvertisementData.makeUrlBuffer called with: ${name}`);
+  
+  var encodedName = Buffer.from(name);
+  if (encodedName.length > MAX_URL_LENGTH) {
+    throw new Error(`Encoded Name must be less than ${MAX_URL_LENGTH} bytes.` +
+                    ` It is currently ${encodedName.length} bytes.`);
+  }
+
+  var fatBeaconHeader = Buffer.alloc(3);
```

Comment:
nit: I still think using `Buffer.from` would be prettier, since you already have the values available.